src/Components/Search/SearchBar.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Date from './Components/Date';
+import Passenger from './Components/Passenger';
+import Button from '../Button';
+import Location from './Components/Location';
+
+function SearchBar({ isRoundTrip }) {
+  const [isModalOn, setIsModalOn] = useState({
+    departure: false,
+    arrival: false,
+    date: false,
+    passenger: false,
+  });
+  const [departure, setDeparture] = useState({
+    name: '서울숲',
+    code: 'SSS',
+  });
+  const [arrival, setArrival] = useState({
+    name: '',
+    code: '',
+  });
+  const [passenger, setPassenger] = useState(1);
+  const [seat, setSeat] = useState('일반석');
+
+  const openModal = e => {
+    const { name } = e.target;
+    setIsModalOn(prev => ({ ...!prev, [name]: true }));
+  };
+
+  const closeModal = name => {
+    setIsModalOn(prev => ({ ...prev, [name]: false }));
+  };
+
+  const switchSpot = async e => {
+    e.preventDefault();
+    const prevArrival = await arrival;
+    setDeparture(prevDeparture => {
+      setArrival(prevDeparture);
+      return prevArrival;
+    });
+  };
+
+  const add = e => {
+    e.preventDefault();
+    setPassenger(prev => ++prev);
+  };
+
+  const substract = e => {
+    e.preventDefault();
+    passenger > 0 && setPassenger(prev => --prev);
```

Comment:
```suggestion
    if (passenger > 0) {
      setPassenger(prev => --prev);
    }
```
값을 리턴하는 것이 아니라 코드의 논리적인 분기를 만들 때는 if 문이 더 좋아보입니다. 한줄 이상의 로직으로 수정해야 할 수도 있으므로 유지 보수 측면에서도 바꾸는게 좋을 것 같습니다!